colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -0,0 +1,351 @@
+        "id": "AyANijPlIN5m"
+      },
+      "source": [
+        "# Update an Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "e4psfYzGeAVS"
+      },
+      "source": [
+        "You can pass new values to update the `description`, `metadata`, and `alias` of an artifact. Call the `save()` method to update the artifact on the W&B servers. You can update an artifact during or outside a W&B Run.\n",
+        "\n",
+        "\n",
+        "This example changes the `description` of the `my_first_artifact` artifact inside a run:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "16fCvZUZJD29"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\")\n",
+        "artifact.description = \"This is an edited description.\"\n",
+        "artifact.save()\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "o-rcMsZYh0Ig"
+      },
+      "source": [
+        "# Download an Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "B2ssTI7LrHS8"
+      },
+      "source": [
+        "To retrieve the path of an Artifact for external use, use the `artifact.download()` method. This will retrieve the directory of an Artifact you select."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "18-7Kd9Oh8cl"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\")\n",
+        "# downloads the artifact to a directory on the W&B platform.\n",
+        "datadir = artifact.download()\n",
+        "run.finish()\n",
+        "# prints the current artifact directory\n",
+        "print(u'\\u2500' * 10)\n",
+        "print(\"Data directory located at:\" + datadir)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "sxox0H9grrcx"
+      },
+      "source": [
+        "For more information on ways to customize your Artifact download, see the [Download and Usage guide](https://docs.wandb.ai/guides/artifacts/download-and-use-an-artifact)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Navigate the Artifacts UI"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "You can also manage your Artifacts via the W&B platform. This can give you insight into your model's performance or dataset versioning. To navigate to the relevant informationm, click this [link](https://wandb.ai/wandb/artifact-basics/overview), then click on the **Artifacts** tab."
```

Comment:
```suggestion
        "You can also manage your Artifacts via the W&B platform. This can give you insight into your model's performance or dataset versioning. To navigate to the relevant information, click this [link](https://wandb.ai/wandb/artifact-basics/overview), then click on the **Artifacts** tab. 
        
        Navigating to the **lineage** section will show you the dependency graph (represented by a DAG) formed by calling run.use_artifact() when an artifact is an input to a run, and run.log_artifact() when an artifact is output to a run. This is a great visualization of the relationship between different model versions and other objects like datasets, jobs, etc. in your project. You will be able to see which runs consumed (run.use_artifact) or produced (run.log_artifact) different model versions. Click [this](https://wandb.ai/wandb/artifact-basics/artifacts/dataset/my_first_artifact/v0/lineage) link to navigate to the project's lineage page."
```